app/src/main/java/io/scalac/degree/data/schedule/filter/ScheduleFilterManager.java

```diff
@@ -168,4 +161,21 @@ public void clearFilters() {
         }
         return result;
     }
+
+    private void setAllFiltersEnabled(boolean enabled) {
+        final Realm realm = realmProvider.getRealm();
```

Comment:
Chyba nie, ja to refactorowałem bo metody clearFilters(), defaultFilters() różniły się tylko boolem.